File: 백준/Bronze/17588. Missing Numbers/Missing Numbers.js
const input = require('fs').readFileSync(0).toString().trim().split('\n');
const n = Number(input.shift());
const numbers = input.map(Number);

const numSet = new Set(numbers);
const last = numbers[numbers.length - 1];

const missing = [];
for (let num = 1; num < last + 1; num++)  {
    if (!numSet.has(num)) {
        missing.push(num);
    }
}

if (missing.length > 0) {
    console.log(missing.join('\n'));
} else {
    console.log('good job');
}

File: 백준/Bronze/17588. Missing Numbers/Missing Numbers.py
n = int(input())
numbers = [int(input()) for _ in range(n)]
num_set = set(numbers)
last = numbers[-1]

missing = []
for num in range(1, last + 1):
    if num not in num_set:
        missing.append(num)

if missing:
    print('\n'.join(map(str, missing)))
else:
    print('good job')